### PR TITLE
Adding Docsettings File. Adding readme presence analysis to  build.

### PR DESCRIPTION
**Pull request by scbedd**
@Azure/azure-sdk-eng 

Issue for `sudo` https://github.com/Azure/azure-sdk/issues/196#issue-410919864

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: .azure-pipelines/client.yml [hunk 1]
```diff
@@ -57,6 +57,13 @@ jobs:
      pip install wheel Jinja2
     displayName: 'Prep Environment'
 
+    # sudo is required for pip install, as a console_entrypoint shim needs to be placed in bin
```
Comment (weshaggard) [inline .azure-pipelines/client.yml]: :( boo.

File: .docsettings.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# make a decision as to what sort of settings file should be used
```
Comment (weshaggard) [inline .docsettings.yml]: Does it make to put this file somewhere other then the root directory here? I know when you get to .NET and Java I'm going to ask you to put it under the "eng" directory.

File: .azure-pipelines/client.yml (hunk 1 above)
Comment (scbedd) [reply .azure-pipelines/client.yml]: Yeah. Thought the same.

File: .docsettings.yml (hunk 2 above)
Comment (scbedd) [reply .docsettings.yml]: Ok.

Comment (scbedd) [reply .docsettings.yml]: Already supported and done in the Java repo.

**Comment by scbedd** [comment]
Pending deploy to pypi such that `doc-warden` will actually install.

**Pull request merged**